notebooks/sentiment_analysis.py

```diff
@@ -0,0 +1,338 @@
+
+The model is `nlptown/bert-base-multilingual-uncased-sentiment` a BERT checkpoint fine-tuned on ~700k reviews in 6 languages and predicts between 1 and 5 stars for each prompt.
+"""
+model_name = "nlptown/bert-base-multilingual-uncased-sentiment"
+multilingual_model = pipelines.pipeline(
+    model=model_name, ipu_config_kwargs=inference_config
+)
+print(simple_test_data)
+multilingual_model(simple_test_data)
+
+"""
+It successfully classifies our simple input, now let's see how it fares with our ambiguous input:
+"""
+multilingual_model([
+    "How are you today?",
+    "I'm a little tired, I didn't sleep well, but I hope it gets better"
+])
+
+"""
+While it is a bit optimistic about our first prompt, it's guess is given with a fairly low confidence, and it identifies the second prompt as neutral with a median score of 3.
+
+Now let's translate our prompts and ask it the same questions in French:
+"""
+ambiguous_in_french = [
+    "Comment vas-tu aujourd'hui?",
+    "Je suis un peu fatigue, je n'ai pas bien dormi mais j'espere que la journee s'ameliore",
+]
+multilingual_model(ambiguous_in_french)
+
+
+"""
+The model gives very similar results in both languages!
+
+Now let us revisit our first model, could it also work with this multi-lingual input?
+"""
+print(sentiment_pipeline(ambiguous_prompts))
+print(sentiment_pipeline(ambiguous_in_french))
+
+"""
+Unsurprisingly it cannot, its prediction are not stable across the two languages. However this model did not perform very well on the ambiguous prompts. If we re-use some of the more precise English-only models and run them on more obvious prompts: 
+"""
+simple_french_input = [
+    "Ce film est excellent",  # This film is excellent
+    "Le produit ne marche pas du tout"  # The tool does not work at all
+]
+print(newer_model(simple_french_input))
+print(multilingual_model(simple_french_input))
+
+"""
+In this case we see the multi-lingual model correctly predict strongly positive and negative labels, while the other model predicts a positive message (correct) and a neutral (expected LABEL_0).
+"""
+"""
+### Other models
+
+Models can be finetuned to extract different classes from text. The `bhadresh-savani/distilbert-base-uncased-emotion` checkpoint is a DistilBERT checkpoint tuned to identify the emotion associated with a prompt:
+"""
+model_name = "bhadresh-savani/distilbert-base-uncased-emotion"
+emotion_model = pipelines.pipeline(model=model_name, ipu_config_kwargs=inference_config)
+emotion_model(simple_test_data)
+
+"""
+This model can be prompted with sentences which include different emotions:
+"""
+emotion_model([
+    "How are you today?",
+    "Don't make me go out, it's too cold!",
+    "What is happening, I don't understand",
+    "Where did you come from?",
+])
+
+
+"""
+## Managing resources
```

Comment:
I'd consider moving this in a separate notebook ("managing IPU resources from notebooks")

Reply:
Good shout, like that we can point to it from different notebooks.